src/main/java/seedu/address/storage/JsonAdaptedGood.java

```diff
@@ -56,7 +60,14 @@ public Good toModelType() throws IllegalValueException {
             throw new IllegalValueException(GoodQuantity.MESSAGE_CONSTRAINTS);
         }
         final GoodQuantity modelGoodQuantity = new GoodQuantity(goodQuantity);
-        return new Good(modelGoodName, modelGoodQuantity);
+
+        if (!GoodQuantity.isValidGoodQuantity(String.valueOf(threshold))) {
+            throw new IllegalValueException(GoodQuantity.MESSAGE_CONSTRAINTS);
+        }
+
```

Comment:
The previous blocks did not have this space (line 63), right?

Reply:
your eye is really sharp